contact.html

```diff
@@ -0,0 +1,367 @@
+                                    <div class="col-lg-12">
+                                        <div class="form-group mb-20">
+                                            <select name="option" class="form-select">
+                                                <option value="how can we help" selected>How can we help you?</option>
+                                                <option value="option 1">option 1</option>
+                                                <option value="option 2">option 2</option>
+                                            </select>
+                                        </div>
+                                    </div>
+                                    <div class="col-lg-12">
+                                        <div class="form-group">
+                                            <textarea rows="10" name="message" class="form-control" placeholder="How can we help you?"></textarea>
+                                        </div>
+                                    </div>
+                                    <div class="col-lg-12 text-center">
+                                        <div class="form-check d-inline-flex mt-30 mb-30">
+                                            <input class="form-check-input me-2 mt-0" type="checkbox" value="" id="flexCheckDefault">
+                                            <label class="form-check-label small" for="flexCheckDefault">
+                                                By submitting, i’m agreed to the <a href="#" class="text-decoration-underline">Terms & Conditons</a>
+                                            </label>
+                                        </div>
+                                    </div>
+                                    <div class="col-lg-12 text-center">
+                                        <input type="submit" value="Send Your Request" class="btn rounded-pill blue5-3Dbutn hover-blue2 sm-butn fw-bold text-light">
+                                    </div>
+                                </div>
+                            </form>
+                        </div>
+                    </div>
+                    <img src="assets/img/icons/contact_a.png" alt="" class="contact_a">
+                    <img src="assets/img/icons/contact_message.png" alt="" class="contact_message">
+                </div>
+            </div>
+        </section>
+        <!-- ====== end contact page ====== -->
+
+        <!-- ====== start contact page ====== -->
+        <div class="map">
+            <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d24219.60999175365!2d-73.9764341314902!3d40.64198229194528!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89c25b400c94a227%3A0x18e2a4d3fb21f0ec!2sFlatbush%2C%20Brooklyn%2C%20NY%2C%20USA!5e0!3m2!1sen!2seg!4v1651361759450!5m2!1sen!2seg" height="500" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>

```

Comment:
Just need to update this to the Aicta's address.

Let me know if you need help for this one. (I have a pretty good idea of what to do for this one)